user/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from .views import CustomLogoutView, CustomLoginView, UserAccountView
+from .views import SignUpView, HomeView
+
+urlpatterns = [
+
+    path('logout/', CustomLogoutView.as_view(), name='logout'),
+    path('', HomeView.as_view(), name='index'),
```

Comment:
HomeView raczej nie jest czescia Usera, tylko czescia glownej apliakcji. Bedziesz tam pewnie wyswietlac paczkomaty czy przyciski do wysylki paczki itd. Tak czy siak nie jest to czesc zwiazana z Uzytkownikiem